omnibus/config/software/vc_redist_14.rb

```diff
@@ -0,0 +1,16 @@
+# Microsoft Visual C++ redistributable
+
+name "vc_redist_140"
```

Comment:
nit: everything's named 140 but you're actually downloading 141

Reply:
it's named 140 because the actual dll we were looking for is vcruntime140.dll (contained in the 141 MSM). I don't mind renaming it, but I think both ways are ambiguous

Reply:
I see! I agree: both ways are ambiguous, forget my comment :)